msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"POT-Creation-Date: 2024-08-18 09:06+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI +ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0\n"
"Language-Code: en\n"
"Language-Name: English\n"
"Preferred-Encodings: utf-8 latin1\n"
"Domain: DOMAIN\n"

#: ../validators/IdValidator.py:26
msgid "Id $id is already in use"
msgstr "Id $id đang được sử dụng"

#: ../validators/IdValidator.py:17
msgid "Spaces are not allowed in ids"
msgstr "Id không được chứa khoảng trắng"

#: ../validators/RegexValidator.py:46
msgid "Validation failed($name): $value of type $type, expected 'string'"
msgstr "Xác thực thất bại ($name): $value thuộc kiểu $type, yêu cầu phải là kiểu chuỗi"

#: ../validators/RegexValidator.py:65
msgid "Validation failed($name): '$value' $errmsg"
msgstr "Xác thực thất bại ($name): '$value' $errmsg"

#: ../validators/EmptyValidator.py:45
msgid "Validation failed($name): '$value' is not empty."
msgstr "Xác thực thất bại ($name): '$value' không rỗng."

#: ../validators/RangeValidator.py:41
msgid "Validation failed($name): '$value' out of range($min, $max)"
msgstr "Xác thực thất bại ($name): '$value' nằm ngoài giới hạn ($min, $max)"

#: ../validators/SupplValidators.py:103
msgid "Validation failed($name): could not convert $value to a date."
msgstr "Xác thực thất bại ($name): không thể chuyển giá trị $value sang ngày tháng."

#: ../validators/RangeValidator.py:33
msgid "Validation failed($name): could not convert '$value' to number"
msgstr "Xác thực thất bại ($name): không thể chuyển giá trị '$value' sang số"

#: ../validators/SupplValidators.py:94
msgid "Validation failed($name): value is empty ($value)."
msgstr "Xác thực thất bại ($name): giá trị rỗng ($value)."

#: ../validators/SupplValidators.py:71
msgid "Validation failed($name: Uploaded data is too large: ${size}MB (max ${max}MB)"
msgstr "Xác thực thất bại ($name: Dữ liệu tải lên quá lớn: ${size}MB (tối đa ${max}MB)"

#: ../validators/BaseValidators.py:51
msgid "contains unprintable characters"
msgstr "có chứa các ký tự không thể hiển thị ra được"

#: ../validators/BaseValidators.py:37
msgid "is not a decimal number."
msgstr "không phải là số thập phân."

#: ../validators/BaseValidators.py:95
msgid "is not a valid email address."
msgstr "không phải là địa chỉ thư điện tử hợp lệ"

#: ../validators/BaseValidators.py:74
msgid "is not a valid international phone number."
msgstr "không phải là số điện thoại quốc tế hợp lệ."

#: ../validators/BaseValidators.py:88
msgid "is not a valid url."
msgstr "không phải là url hợp lệ"

#: ../validators/BaseValidators.py:66
msgid "is not a valid us phone number."
msgstr "không phải là số điện thoại của Mỹ hợp lệ."

#: ../validators/BaseValidators.py:81
msgid "is not a valid zip code."
msgstr "không phải là zip code hợp lệ"

#: ../validators/BaseValidators.py:58
msgid "is not a well formed SSN."
msgstr "không phải là số SSN đúng định dạng."

#: ../validators/BaseValidators.py:44
msgid "is not an integer."
msgstr "không phải là số nguyên."

#: ../validators/BaseValidators.py:109
msgid "this name is not a valid identifier"
msgstr "đây không phải là một định danh hợp lệ"

#: ../validators/ExpressionValidator.py:94
msgid "validation failed, expr was:$expr"
msgstr "xác thực thất bại, thông điệp trả về: $expr"
